O/ODEInterface/build_tarballs.jl

```diff
@@ -0,0 +1,329 @@
+using BinaryBuilder
+
+name = "ODEInterface"
+version = v"0.0.1"
+
+function get_autogenerated_build_script()
```

Comment:
Any particular reason why you have a function which returns a constant string?  Can't it just be a string, as usual?

Reply:
I can make a static string. I used the function name to try to indicate that this bash-script was auto-generated by the package (`ODEInterface`) itself. I can change this to a comment (or even delete the comment) if this helps.